### PR TITLE
add ssl_verification_enabled to streaming_specs + tests

### PR DESCRIPTION
**Pull request by BRONSOLO**
ping @chriddyp @theengineear please review. Addresses: https://github.com/plotly/plotly.py/issues/465

- [x] update version (minor?)
- [x] add note in changelog for python 2.7.9/3.4.3 requirement for streaming over SSL.

### CODE REVIEW

**Comment by theengineear** [comment]
>update version (minor?)

👍 

💃

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [1.11.0] - 2016-06-06
```
Comment (chriddyp) [inline CHANGELOG.md]: you'll need to bump this up to `1.11.1` to upgrade it on `pip`

Comment (BRONSOLO) [inline CHANGELOG.md]: Oh yep - sorry, copy and paste : ) I'll bump it to `1.12.0`

**Comment by chriddyp** [comment]
💃

**Pull request merged**